### PR TITLE
Adding 2 DoodleNet examples

### PR DESCRIPTION
**Pull request by yining1023**
## → Submit changes to the relevant branch 🌲
`development`

## → Description 📝
- submitting a new feature 🆕 

## → Screenshots 🖼
Screen Shot 2019-05-25 at 9 47 09 PM
Screen Shot 2019-05-25 at 9 51 49 PM

## → Relevant documentation 🌴
This PR adds two `ml5.imageclassifier('DoodleNet')` examples - 
- DoodleNet with canvas
- DoodleNet with webcam video(The doodle needs to be held closely to the webcam with good lighting to have a good result :))

### CODE REVIEW

**Comment by joeyklee** [comment]
Thanks so much @yining1023 ! I will test tomorrow, but the examples you made look otherwise awesome. So concise and elegant. @shiffman might have some thoughts on using the `.map()` function, but otherwise, wonderful to see the doodlenet in action!

File: p5js/ImageClassification/ImageClassification_DoodleNet_Canvas/sketch.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+// A function to run when we get any errors and the results
+function gotResult(error, results) {
+  // Display error in the console
+  if (error) {
+    console.error(error);
+  }
+  // The results are in an array ordered by confidence.
+  console.log(results);
+  // Get all top 10 labels
+  const labels = results.map(r => r.label);
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_DoodleNet_Canvas/sketch.js]: `map()` might be unfamiliar to beginners, maybe just do something simpler like show the first label and confidence?

File: p5js/ImageClassification/ImageClassification_DoodleNet_Canvas/sketch.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+function setup() {
+  // Create a canvas with 280 x 280 px
+  canvas = createCanvas(280, 280);
+  // Set canvas background to white
+  background(255);
+  // Whenever mouseReleased event happens on canvas, call "classifyCanvas" function
+  canvas.mouseReleased(classifyCanvas);
+  // Create a clear canvas button
+  createClearBtn();
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_DoodleNet_Canvas/sketch.js]: you could probably just include the code for these DOM elements right here in `setup()` for simplicity?

**Review by shiffman — COMMENTED** [review]
This is so so amazing, thank you @yining1023!

**Comment by shiffman** [comment]
haha @joeyklee read my mind!

**Comment by yining1023** [comment]
haha. Thank you both for the suggestions!
I made some updates -
- Only showing the first label and confidence
- Moved the DOM elements creation to `setup()`.

**Comment by joeyklee** [comment]
merged! Thanks so much @yining1023 - these are awesome! 

Screen Shot 2019-05-27 at 10 37 07

I got slightly different results for umbrella, but the word association is super interesting umbrella ==> rain 
Screen Shot 2019-05-27 at 10 44 53

**Pull request merged**